scripts/user/add-judge.ts

```diff
@@ -0,0 +1,126 @@
+import { JudgeTitle } from '../../shared/src/business/entities/EntityConstants';
```

Comment:
```Consider adding a shbang to first line of cli scripts, e.g. 
#!/usr/bin/env npx ts-node --transpile-only
```
You can also [set the execute bit](https://ionic.zendesk.com/hc/en-us/articles/360000160067-Adding-execute-permissions-using-git_)

Reply:
Blargh - i hate github sometimes.  I was suggesting adding a line to the top of the script, not replacing the highlighted line.  🤦

Reply:
Got it--thank you!